src/download.ts

```diff
@@ -51,18 +51,18 @@ export async function downloadPiperBinary (
 
   return piperPath
 }
-async function getPiperDownloadURL (piper: string, version: string): Promise<string> {
-  const tagURL = `${GITHUB_COM_SERVER_URL}/SAP/jenkins-library/releases/${getTag(version, false)}`
-  const response = await fetchRetry(tagURL, 'HEAD')
-    .catch(async (err) => {
-      throw new Error(`Can't get the tag: ${err}`)
-    })
-  return await Promise.resolve(response.url.replace(/tag/, 'download') + `/${piper}`)
-}
 
-async function getPiperBinaryNameFromInputs (isEnterpriseStep: boolean, version: string): Promise<string> {
-  if (version === 'master') {
-    info('using _master binaries is deprecated. Using latest release version instead.')
+export async function getPiperDownloadURL (piper: string, version: string): Promise<string> {
+  try {
+    const response = await fetchRetry(getDownloadUrlByTag(version), 'HEAD')
```

Comment:
function name suggest that this is simply building an url, why do we do a fetch here already?